controller/setting_controller.go

```diff
@@ -293,6 +293,11 @@ func (sc *SettingController) updateBackupstorePollInterval() (err error) {
 }
 
 func (bm *BackupStoreMonitor) Start() {
+	if bm.pollInterval == time.Duration(0)*time.Second {
+		bm.Stop()
```

Comment:
Don't need to call this. Before entering this function, the monitor should be stopped already. The `wait.Until` call below is a blocking call.